harmony-kotlin/src/commonTest/kotlin/com/harmony/kotlin/data/datasource/validator/vastra/strategies/timestamp/TimestampValidationStrategyTest.kt

```diff
@@ -41,4 +49,43 @@ class TimestampValidationStrategyTest {
 
     assertFalse(isValid)
   }
+
+  @Test
+  fun should_value_be_invalid_if_value_is_expired_2() {
+    runTest {
+
+      val yesterday = Clock.System.now().minus(1.days).toEpochMilliseconds()
+
+      val foo = Foo("bar", lastUpdatedAt = Clock.System.now().toEpochMilliseconds(), expireIn = 1.minutes.toLong(DurationUnit.MILLISECONDS))
+
+      val foos = listOf(
+        Foo("bar", lastUpdatedAt = Clock.System.now().toEpochMilliseconds(), expireIn = 1.minutes.toLong(DurationUnit.MILLISECONDS)),
+        Foo("bar", lastUpdatedAt = Clock.System.now().toEpochMilliseconds(), expireIn = 1.minutes.toLong(DurationUnit.MILLISECONDS))
+      )
+
+      val timestampValidationStrategy = TimestampValidationStrategy()
+      val validationServiceManager: ValidationService = ValidationServiceManager(listOf(timestampValidationStrategy))
+
+      val ds = InMemoryDataSource<List<Foo>>().also {
+        it.put(KeyQuery("foos"), foos)
+      }
+
+//      val ds = InMemoryDataSource<Foo>().also {
+//        it.put(KeyQuery("foos"), foo)
+//      }
+
+//      val actual = GetDataSourceVastraValidator(
+//        ds, validationServiceManager
+//      ).get(KeyQuery("foos"))
+
```

Comment:
Better remove it if it's not needed.

Reply:
Done. This was a test that was supposed to be removed. I replaced it